contracts/TokenHandler.sol

```diff
@@ -4,22 +4,27 @@ import "../installed_contracts/zeppelin-solidity/contracts/ownership/Ownable.sol
 import "../installed_contracts/zeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol";
 
 contract TokenHandler is Ownable {
-
     address public targetToken;
 
-    constructor ( address _targetToken) public Ownable() {
+    constructor (address _targetToken) public Ownable() {
+        require(_targetToken != 0x0, "Must specify a target token!");
         setTargetToken(_targetToken);
     }
 
-    function getTokenBalance(address _token) public view returns (uint256) {
-        ERC20Basic token = ERC20Basic(_token);
-        return token.balanceOf(address(this));
+    function setTargetToken(address _target)
+        public
+        onlyOwner
+        returns (bool success)
+    {
+        targetToken = _target;
```

Comment:
If we are going to allow overwriting targetToken, we should have a sort of check, especially since the contract is not really owned by just the `owner`, bust instead by the beneficiaries